Client/src/pages/UserDisplay/UserDisplay.jsx

```
@@ -0,0 +1,146 @@
+    phoneNumber: "",
+    dob: "",
+    gender: true, // Mặc định cho giới tính
+  });
+  const [isEditing, setIsEditing] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState("");
+
+  // Hàm định dạng ngày
+  const formatDate = (dateString) => {
+    const date = new Date(dateString);
+    if (isNaN(date)) return ""; // Trả về chuỗi rỗng nếu ngày không hợp lệ
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)